### DISCUSSION TITLE
Actions Feature Request:`milestoned` event for pull requests

### DISCUSSION THREAD

**Discussion by albertvaka** [community/community#12846] [Actions]
Currently, an issue getting assigned a milestone triggers an event (`milestoned`) that can trigger a workflow. However, the same isn't true for PRs.

I think this is an oversight, since similar events like `labeled` are available both for PRs and issues.

Reference:
- [Events for issues](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issues)
- [Events for PRs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)

**Comment by ethomson** [comment]
Hi @albertvaka - as you noted, this is not supported today, per the documentation. I will talk to the team that manages triggers, but at the moment this is working as it's documented. It's possible that we can extend this in the future and if so, I'll make sure that this is on their backlog.

Thanks much for the feedback!

**Comment by bsipocz** [comment]
This would be super useful for us, too to enable milestone-triggered checks for PRs.

**Comment by pllim** [comment]
I have been waiting for this for a few years. Someone else requested this way back when at https://github.community/t/feature-request-add-milestone-changes-as-activity-type-to-pull-request/16778

**Comment by DavidChangTR** [comment]
This has now been updated. Milestoning and demilestoning for a pull request now triggers a pull request event trigger, not issues event trigger. In fact, an issues event trigger does not occur anymore, only pull request event trigger. Figured this out the hard way of debugging literally everything except adding milestoned and demilestoned event triggers into the pull_request event triggers, the 'milestoned' and 'demilestoned' triggers for issues are no longer triggered by a milestone change in a pull request. 

This has not been updated in the official documentation as of me writing this.

**Reply by jsoref** [reply]
Fwiw, I opened https://github.com/github/docs/issues/23909 which may or may not be related. (I can't tell, because, as you and I note, the documentation isn't current.)

**Reply by joshhunt** [reply]
Just to confirm also - the `pull_request_target` event trigger also supports `milestoned` and `demilestoned` from my testing.
